xapian-core/include/xapian/queryparser.h

```diff
@@ -234,6 +234,79 @@ class XAPIAN_VISIBILITY_DEFAULT RangeProcessor
     }
 };
 
+// Handles a file size range
```

Comment:
This comment should also note the form of the values, which is the same as for `NumberRangeProcessor`: they must be serialised appropriately.